src/neuroglancer/webgl/shader.ts

```diff
@@ -105,9 +105,11 @@ export function getShader(gl: WebGLRenderingContext, source: string, shaderType:
       });
       s += `\n</pre>`;
       let w = window.open('about:blank', '_blank');
-      try {
-        w.document.write(s);
-      } catch (writeError) {
+      if (w !== null) {
+        try {
+          w.document.write(s);
+        } catch (writeError) {
+        }
```

Comment:
```
    ERROR in ./src/neuroglancer/webgl/shader.ts
    [tsl] ERROR in /Users/John/2017/fall/neuroglancer/src/neuroglancer/webgl/shader.ts(109,9)
          TS2531: Object is possibly 'null'.
```
Again, I have no idea why `javascript` would refuse to open a tab/window. Maybe it could happen if Neuroglancer is embedded in a sandboxed `iframe`? It's not a priority for me to look into this right now.